src/__tests__/validation/FlowContainerValidator.spec.ts

```diff
@@ -1,332 +1,562 @@
 import {getFlowStructureErrors} from '../../domain/validation/FlowContainerValidator'
 
 describe('Validation Usage Tests', () => {
+  const version = '1.0.0-rc4'
   test('Valid Usage Example', async () => {
     const flowJson = `{
-        "specification_version": "1.0.0-rc2",
-        "uuid": "e0616de8-68c5-4918-b375-f5eb9fcc1695",
-        "name": "Museum of Interop Reception",
-        "description": "Welcome and main menu for the Museum of Interoperability",
-        "flows": [
-          {
-            "uuid": "b15be41c-d29b-41fb-b981-26b2ebe8a6ff",
-            "name": "Museum of Interop Reception",
-            "label": "Welcome and main menu for the Museum of Interoperability",
-            "last_modified": "2021-03-14 16:11:43.326Z",
-            "interaction_timeout": 172800,
-            "supported_modes": [
-              "SMS",
-              "IVR",
-              "USSD"
-            ],
-            "languages": [
-              {
-                "id": "22",
-                "label": "English",
-                "iso_639_3": "eng"
+      "specification_version": "${version}",
+      "uuid": "e0616de8-68c5-4918-b375-f5eb9fcc1695",
+      "name": "Museum of Interop Reception",
+      "description": "Welcome and main menu for the Museum of Interoperability",
+      "flows": [
+        {
+          "uuid": "b15be41c-d29b-41fb-b981-26b2ebe8a6ff",
+          "name": "Museum of Interop Reception",
+          "label": "Welcome and main menu for the Museum of Interoperability",
+          "last_modified": "2021-03-14 16:11:43.326Z",
+          "interaction_timeout": 172800,
+          "supported_modes": [
+            "SMS",
+            "IVR",
+            "USSD"
+          ],
+          "languages": [
+            {
+              "id": "22",
+              "label": "English",
+              "iso_639_3": "eng"
+            },
+            {
+              "id": "23",
+              "label": "Swahili - Kenyan",
+              "iso_639_3": "swa",
+              "variant": "kenyan",
+              "bcp_47": "sw-KE"
+            }
+          ],
+          "vendor_metadata": {},
+          "blocks": [
+            {
+              "uuid": "3d2d806a-19a9-4827-93c3-22d4357ff8b2",
+              "ui_metadata": {
+                "canvas_coordinates": {
+                  "x": 200,
+                  "y": 158
+                }
               },
-              {
-                "id": "23",
-                "label": "Swahili - Kenyan",
-                "iso_639_3": "swa",
-                "variant": "kenyan",
-                "bcp_47": "sw-KE"
+              "type": "MobilePrimitives.Message",
+              "name": "welcome_message",
+              "label": "Welcome Message",
+              "semantic_label": "",
+              "exits": [
+                {
+                  "uuid": "d4cb73df-c993-4395-9f75-1884ab7a0176",
+                  "name": "Default",
+                  "default": true,
+                  "vendor_metadata": {},
```

Comment:
Assuming this should be in the rc3 pr really? I'm happy to leave it here though.

Reply:
Thanks !